dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/h2/V1.2__Initial_DSpace_1.2_Oracle_database_schema.sql

```diff
@@ -102,9 +102,9 @@ CREATE TABLE EPerson
   password            VARCHAR2(64),
   firstname           VARCHAR2(64),
   lastname            VARCHAR2(64),
-  can_log_in          NUMBER(1),
-  require_certificate NUMBER(1),
-  self_registered     NUMBER(1),
+  can_log_in          BOOLEAN,
+  require_certificate BOOLEAN,
+  self_registered     BOOLEAN,
```

Comment:
I'm altering the old files since this is the in memory database (& I hoped this shouldn't matter that much). If you did this in oracle or postgres you would indeed get exceptions.